src/pages/references/stacks-blockchain.md

```diff
@@ -1,19 +1,9 @@
 ---
-title: Stacks Blockchain APIs
-description: Interacting with the Stacks 2.0 Blockchain
+title: Stacks Blockchain API
+description: Interacting with the Stacks 2.0 Blockchain via API
 ---
 
-## Introduction
```

Comment:
I feel like we should keep introducing the differences between node RPC API and our hosted node JSON API I remember that we felt strongly about that when we designed the content. @zone117x to input here as well please :)

Reply:
Added some details, see updated commit